.travis.yml

```diff
@@ -68,8 +68,15 @@ install:
   - travis_retry docker-compose -f docker-compose.deps.yml run --rm pip
   - travis_retry docker-compose -f docker-compose.deps.yml run --rm assets
   - travis_retry docker-compose -f docker-compose.test.yml pull --parallel ${SUITE} ${IMAGES}
+  - if [ ! -z "${IMAGES}" ]; then
+      travis_retry docker-compose -f docker-compose.test.yml build ${IMAGES};
+    fi
```

Comment:
why we need this on ``install`` and ``script``?

Reply:
discussed IRL